src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -236,7 +236,10 @@ describe('processBulkUpload', () => {
 		);
 
 		const updatedBulkUpload = await loadBulkUpload(bulkUpload.id);
-		expect(updatedBulkUpload.status).toEqual(BulkUploadStatus.FAILED);
+		expect(updatedBulkUpload).toMatchObject({
+			status: BulkUploadStatus.FAILED,
+			fileSize: null,
```

Comment:
Ugh, null -- so null is what is being returned by the pg library.  I wish it was `undefined` rather than `null` but... well... so it goes I guess unless someone else has better thoughts

Reply:
As I understand it, we can't add a `NULL` constraint to the database since bulk uploads are created before the `file_size` field is calculated/populated. And so `null` is the correct value to return (it's what's in the database, after all). I think it's fine and have no concerns about this.

Reply:
So as an example of why null is the worst, it looks like maybe null gets turned into 0 when serializing.

I'm inclined to just call that acceptable behavior for now.  The API will return `0` in cases where file size is null.

Reply:
wat why, who does that, who's the culprit

Reply:
Ok honestly this is a good question and I'm gonna investigate as part of the PR!  Marking the rest for re-review at your leisure though.

Reply:
found out: it's ajv -- https://ajv.js.org/coercion.html

I'm going to look into whether I can have it leave null alone.

Reply:
FWIW this is what I was thinking about regarding not wanting to use null: https://github.com/microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined

Reply:
Fixed, making schema "nullable: true" -- though now the type has to be optional? which is somewhat weird (since my undersatnding is `?` allows *undefined* rather than null).